sys/vm/vm_map.c

```diff
@@ -2678,13 +2686,13 @@ vm_map_pmap_enter(vm_map_t map, vm_offset_t addr, vm_prot_t prot,
 			}
 		} else if (p_start != NULL) {
 			pmap_enter_object(map->pmap, start, addr +
-			    ptoa(tmpidx), p_start, prot);
+			    ptoa(tmpidx), p_start, prot, pmap_flags);
```

Comment:
This is the part that probably really matters.  pmap_enter_quick is an optimization that it should have been safe to skip.  However, I don't think this bit of code can be skipped.  Or at least if it was, we weren't skipping it.

Reply:
Yeah, that's what my commit message was trying to say.